game/ui/tileview/citytileview.cpp

```diff
@@ -740,15 +740,21 @@ void CityTileView::render()
 					{
 						case VehicleMission::MissionType::AttackVehicle:
 						{
-							vehiclesUnderAttack.insert(m->targetVehicle);
```

Comment:
Would this be cleaner as:
if (!m->targetVehicle)
    break;

rendering_code();

Reply:
Fair enough, that will also reduce the indentation level